wrangler.example.toml

```diff
@@ -1,7 +1,7 @@
 compatibility_date = "2023-06-21"
-name = "<your name>"
+name = "remix-movies"
 
 [[d1_databases]]
 binding = "DB"
 database_id = "<your database id>"
-database_name = "<your database name>"
+database_name = "remix-movies-db"
```

Comment:
Up to you if you want to avoid hardcoding some values in the example toml (these are the names suggested in the README). My hope was to dummy proof one more thing